src/renderer/screens/onboarding/steps/GenuineCheck/index.js

```diff
@@ -76,12 +81,18 @@ const GenuineCheck = (props: StepProps) => {
     setGenuineCheckModalOpened(false);
   }, []);
 
-  const handleGenuineCheckPass = useCallback(() => {
-    updateGenuineCheck({
-      isDeviceGenuine: true,
-    });
-    setGenuineCheckModalOpened(false);
-  }, [updateGenuineCheck]);
+  const handleGenuineCheckPass = useCallback(
+    ({ result }: any) => {
+      if (!hasCompletedOnboarding && (!result || !result.installed.length)) {
+        dispatch(setHasInstalledApps(false));
```

Comment:
ok so to my understanding you do not need to do this because it will happen anyway in the new `useEffect` inside AppsList. right?